src/CarbonAware.Aggregators/src/SciScore/SciScoreAggregator.cs

```diff
@@ -29,11 +33,12 @@ public SciScoreAggregator(ILogger<SciScoreAggregator> logger, ICarbonIntensityDa
     }
 
     /// <inheritdoc />
+    [ObsoleteAttribute("This method is obsolete. Use ICarbonAwareAggregator equivalent method instead.", false)]
```

Comment:
Question: Why keeping the classes related to `SciScoreAggregator`, when the `SciScoreController` is consuming now `CarbonAwareAggregator`? Would it be simpler to remove those?

Reply:
well, from "pre-review" chats with @bderusha, the plan decided on was to "not delete" the code with this PR, and leave "full" clean-up of Sci-related-areas for sharing phase ...